docs/modules/ROOT/pages/ai-services.adoc

```diff
@@ -141,34 +141,26 @@ We'll explore an alternative approach to avoid manual memory handling in the <<m
 
 While LLMs are the base AI models, the chat language model builds upon them, enabling chat-like interactions. If you have a single chat language model, no specific configuration is required.
 
-However, with multiple models like OpenAI and Hugging Faces, you need to specify which one to use:
+However, when multiple model providers are present in the application (such as OpenAI, Azure OpenAI, HuggingFace, etc.)
+each model different model needs to be given a name, which is then referenced by the AI service like so:
 
 [source,java]
 ----
-@RegisterAiService(
-    chatLanguageModelSupplier = MyChatModelSupplier.class
-)
+@RegisterAiService(modelName="m1")
```

Comment:
Now that we have the qualifier, should it be used there instead of a parameter? Or allow both?

Reply:
I don't really like having multiple unless we really have to